tests/Colder.Tests/InMemoryDistributedLockTest.cs

```diff
@@ -49,7 +50,7 @@ public async Task Test()
                 await (await aTask);
             }
 
-            Assert.AreEqual(num, 160000);
+            Assert.AreEqual(1600, num);
```

Comment:
Interlocked is thread safe,but this code is to test the distributedlock is work fine or not,so i think there is unnecessary